### ISSUE TITLE
Update REXML to fix DoS Vulnerability (CVE-2024-35176)

### ISSUE THREAD

**Issue by fchiusolo** [CocoaPods/Xcodeproj#947]
Hi team,

There's a DoS vulnerability in `rexml` before version 3.2.7. It affects `xcodeproj` through `fastlane`. Can you update `rexml` to version 3.2.7 or later?

More details: [ruby-lang.org](https://www.ruby-lang.org/en/news/2024/05/16/dos-rexml-cve-2024-35176/).

Thanks!

**Comment by bmedenwald** [comment]
It's worse now. New CVE: https://www.ruby-lang.org/en/news/2024/07/16/dos-rexml-cve-2024-39908/

This needs to be resolved.

**Comment by bmedenwald** [comment]
Added PR: https://github.com/CocoaPods/Xcodeproj/pull/948

**Comment by samfranz** [comment]
When can we expect a release with this change?

**Comment by elkinjosetm** [comment]
The PR was already merged, any idea when should we expect to have a new release?

**Comment by sphanley** [comment]
Seconding the request for a release at the earliest convenience– my enterprise team is running into security warnings related to this rexml dependency, so it would be hugely helpful to have a new version released with the recently updated constraint.

**Comment by sphanley** [comment]
For others impacted by these security warnings in the short term, if you're comfortable being "between releases", you can target the current commit by adding this to your Gemfile: 

> gem "xcodeproj",  git: "https://github.com/CocoaPods/Xcodeproj.git", :branch => "master", ref: "fe55cf5"

Since the Version defined in the repo is still 1.24.0, this will let you consume the updated dependency list while still satisfying transitive dependencies on 1.24.0.

**Comment by kconner** [comment]
Please release a numbered version with the merged dependency update. The latest version of this library, 1.24.0, is still vulnerable.

**Comment by konrad-gibaszewski** [comment]
Please release a numbered version with the merged dependency update.

**Comment by 0rax** [comment]
For anybody still struggling with this, you can point your `Gemfile` to this git repository directly to retrieve the latest version from master.

This worked fine in my case:

```ruby
source 'https://rubygems.org'

ruby '>= 2.6.10'

gem 'cocoapods', '>= 1.15.2'
gem 'xcodeproj', '~> 1.24', git: 'https://github.com/CocoaPods/Xcodeproj.git'
gem "rexml", "~> 3.3.2"
```

Once a new version is release, just remove the `git:` part and update the version identifier next to `xcodeproj`.

**Comment by Aaron-Empower** [comment]
Unfortunately, many of us are exposed to this vulnerability through Fastlane's use of xcodeproj as a dependency.

Looking forward to the next numbered release with bated breath!

**Comment by Kaspik** [comment]
@amorde Hola Eric! Is there a chance we could get new release out asap?

**Comment by amorde** [comment]
The fix for this was released in [1.25.0](https://github.com/CocoaPods/Xcodeproj/releases/tag/1.25.0).
